cra-tf-validate-ignore-rules.json

```diff
@@ -1,3 +1,10 @@
 {
-    "scc_rules": []
+    "scc_rules": [
+        {
+            "scc_rule_id": "rule-c97259ee-336d-4c5f-b436-1868107a9558",
+            "description:": "Check whether Cloud Object Storage is enabled with customer-managed encryption and Keep Your Own Key (KYOK)",
+            "ignore_reason": "This module does not create any Cloud object storage and it is used in an example for testing purpose.",
```

Comment:
This is not correct - the Da is creating a COS bucket and it supports HPCS encryption